policies/allowed_vm_sku/policy.json

```diff
@@ -0,0 +1,52 @@
+{
+  "properties": {
+    "displayName": "HMCTS Allowed VM SKU sizes",
+    "policyType": "BuiltIn",
+    "mode": "Indexed",
+    "parameters": {
+      "allowedSkus": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Allowed SKU names",
+          "description": "HMCTS allowed VM SKU sizes",
+        },
+        "allowedValues": [
+          "Standard_D2ds_v5",
+          "Standard_D4ds_v5",
+          "Standard_DD16ds_v5",
+          "Standard_FD16ds_v5",
+          "Standard_E2ds_v5"
+        ],
+        "defaultValue": [
+          "Standard_D2ds_v5",
+          "Standard_E2ds_v5"
+        ]
```

Comment:
...for setting defaults maybe add all items to list as well, this makes it also available from the dropdown in the portal ui

Reply:
included all sizes as default